src/Stratis.Bitcoin/Utilities/NodeStats.cs

```diff
@@ -35,6 +40,7 @@ public NodeStats(IDateTimeProvider dateTimeProvider)
             this.locker = new object();
             this.dateTimeProvider = dateTimeProvider;
 
+            this.nodeBenchmarks = new List<Func<INodeBenchmark>>();
```

Comment:
there is already a stats collection that has benchmarks. why new collection is needed?